project_euler/problem_99/sol1.py

```diff
@@ -0,0 +1,33 @@
+"""
+Problem:
+
+Comparing two numbers written in index form like 2'11 and 3'7 is not difficult, as any calculator would confirm that 2^11 = 2048 < 3^7 = 2187.
+
+However, confirming that 632382^518061 > 519432^525806 would be much more difficult, as both numbers contain over three million digits.
+
+Using base_exp.txt, a 22K text file containing one thousand lines with a base/exponent pair on each line, determine which line number has the greatest numerical value.
+
+NOTE: The first two lines in the file represent the numbers in the example given above.
+"""
+
+import os
+from math import log10
+
+
+def find_largest() -> int:
```

Comment:
Let’s change the function signature to allow users to experiment with different datasets:
`def find_largest(data_file: str= "base_exp.txt") -> int:`